File: training-data/go/rotation_check.go
package main

import (
	"fmt"
	"strings"
)

func isRotation(a, b string) bool {
	return len(a) == len(b) && strings.Contains(a+a, b)
}

func main() {
	fmt.Println(isRotation("waterbottle", "erbottlewat"))
	fmt.Println(isRotation("abc", "acb"))
}

File: training-data/rust/rotation_check.rs
fn is_rotation(a: &str, b: &str) -> bool {
    a.len() == b.len() && a.repeat(2).contains(b)
}

fn main() {
    println!("{}", is_rotation("waterbottle", "erbottlewat"));
    println!("{}", is_rotation("abc", "acb"));
}
